tin/apps/assignments/models.py

```diff
@@ -50,19 +67,34 @@ def filter_permissions(self, user, *perms: Literal["-", "r", "w"]):
             return self.filter(q).distinct()
 
     def filter_visible(self, user):
+        r"""Filters assignments that are visible to a user
+
+        Alias for calling :meth:`filter_permissions` with the permissions
+        "r" and "w"
+        """
         return self.filter_permissions(user, "r", "w")
 
     def filter_submittable(self, user):
+        """Filters by assignments that can be submitted.
+
+        .. warning::
+
+            Do NOT use this if :attr:`~Assignment.is_quiz` is ``True``.
+            In that case, the check should be done manually.
+        """
         return self.filter_permissions(user, "w")
 
     def filter_editable(self, user):
+        """Filters assignments if they're editable by the user"""
         if user.is_superuser:
             return self.all()
         else:
             return self.filter(course__teacher=user).distinct()
 
 
+# FIXME: Why is there an empty unused argument, should this be a method of assignment?
```

Comment:
The unused argument is there because this function is used as the `grader_file` `FileField` `upload_to` function in line 155.